python/pylibcudf/pylibcudf/io/orc.pyi

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from typing import Any
+
+from pylibcudf.io.types import SourceInfo, TableWithMetadata
+from pylibcudf.types import DataType
+
+def read_orc(
+    source_info: SourceInfo,
+    columns: list[str] | None = None,
+    stripes: list[list[int]] | None = None,
+    skip_rows: int = 0,
+    nrows: int = -1,
+    use_index: bool = True,
+    use_np_dtypes: bool = True,
+    timestamp_type: DataType | None = None,
+    decimal128_columns: list[str] | None = None,
+) -> TableWithMetadata: ...
+
+class OrcColumnStatistics:
+    @property
+    def number_of_values(self) -> int | None: ...
+    @property
+    def has_null(self) -> bool | None: ...
+    def __getitem__(self, item: str) -> Any: ...
+    def __contains__(self, item: str) -> bool: ...
+    def get[T](self, item: str, default: None | T = None) -> T | None: ...
```

Comment:
This is a py-3.11ism, but because it's only observed by the type checker that's fine.
```suggestion
    def get[T](self, item: str, default: T | None = None) -> T | None: ...
```